Lib/sqlite3/test/types.py

```diff
@@ -313,19 +313,19 @@ def tearDown(self):
         self.cur.close()
         self.con.close()
 
-    def CheckCursorDescriptionCTESimple(self):
+    def test_cursor_description_cte_simple(self):
         self.cur.execute("with one as (select 1) select * from one")
         self.assertIsNotNone(self.cur.description)
         self.assertEqual(self.cur.description[0][0], "1")
 
-    def CheckCursorDescriptionCTESMultipleColumns(self):
+    def test_cursor_description_ctes_multiple_columns(self):
```

Comment:
```suggestion
    def test_cursor_description_cte_multiple_columns(self):
```

@berkerpeksag: ctes => cte?

Reply:
CTE sounds good to me.